.github/workflows/cd.yml

```diff
@@ -0,0 +1,93 @@
+name: CD
+
+on:
+  push:
+    tags-ignore:
+      - "*.dev0"
+  pull_request:
+    branches:
+      - develop
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  Build:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        include:
+          - arch: x64
+            pythonVersion: 3.7
+            gvsbuild: "https://github.com/deluge-torrent/gvsbuild-release/releases/download/latest/gvsbuild-py3.7-vs16-x64.zip"
```

Comment:
Easier to maintain if move `gvsbuild` as an env to install deps section, use python and arch to populate filename.